Assets/PatchKit/UnityEditorExtension/Editor/UI/SafeBuildAndUploadScreen.cs

```diff
@@ -436,6 +445,16 @@ private void BuildAndUpload()
         Assert.IsNull(VersionLabelValidationError);
         Assert.IsNull(VersionChangelogValidationError);
 
+        if (Core.Api.GetAppInfo(new AppSecret(_appSecret)).Removed)
+        {
+            EditorUtility.DisplayDialog(

```

Comment:
Displaying this dialog should be moved to Initialize function. We don't need to double-check if app has been removed, that's not very crucial.

Reply:
if the editor extension is open with application and this application has been removed, the Initialize function will not be executed and the user may click BuildAndUpload. So i think we need this double-check

Reply:
The probability of that is very low. Besides, we need to show that dialog before unlinking the app, so it has to be moved to the Initialize method.

Reply:
done